JSTests/ChakraCore/test/es5/ObjLitGetSet.baseline-jsc

```diff
@@ -21,6 +21,6 @@ Success: Expected invalid syntax if set keyword is a string (then it is just a n
 Success: Expected invalid syntax if get keyword is a string (then it is just a normal property name) = true, got true
 PASS
  (test 5): Object literal get set function toString
-function () { return _foo; }
-function (value) { _foo = value; }
+get foo() { return _foo; }
+set foo(value) { _foo = value; }
```

Comment:
Those two things are outstanding (and it's not even the Chakra baseline) but as I understand the ECMA spec, this is what it should be. See

JSTests/test262/test/built-ins/Function/prototype/toString/getter-object.js